src/proto/grpc/testing/messages.proto

```diff
@@ -38,9 +38,6 @@ package grpc.testing;
 enum PayloadType {
   // Compressable text format.
   COMPRESSABLE = 0;
-
-  // Uncompressable binary format.
-  UNCOMPRESSABLE = 1;
```

Comment:
If we're doing that, then why don't we trash PayloadType altogether? If we think we may need PayloadType, then keeping uncompressable may be prudent.

But in general I'd prefer to get it removed from the code before removing it from the proto (including internally).